src/theme/base/_fonts.scss

```diff
@@ -0,0 +1,26 @@
+/* open-sans-regular - cyrillic-ext_vietnamese_cyrillic_greek_greek-ext_latin_latin-ext */
```

Comment:
If the fonts are already defined in the `public/index.html` file, is it necessary to have this file?

Reply:
It is currently there if you need this instead of the "critical CSS" approach to prevent FOIT and FOUT.

But it is not actively used as the import to this file is commented out.